CONTRIBUTING.md

```diff
@@ -206,6 +215,22 @@ Run `./gradlew :app:recordScreenshots` to capture all screenshots. Files will be
 
 > Note: Python is required.
 
+The generated screenshots will be saved in the component folder in `docs` directory. To include it to readme, you can use the following syntax (for Compose, for View replace the path):
```

Comment:
also clarify the `default` naming please + how spaces etc are formatted

Reply:
Done